prepare_vscode.sh

```diff
@@ -8,10 +8,15 @@ cd vscode || exit
 ../update_settings.sh
 
 # apply patches
-patch -u src/vs/platform/update/electron-main/updateService.win32.ts -i ../patches/update-cache-path.patch
-patch -u resources/linux/rpm/code.spec.template -i ../patches/fix-rpm-spec.patch
-git apply --ignore-whitespace ../patches/binary-name.patch
-git apply --ignore-whitespace ../patches/custom-gallery.patch
+for file in ../patches/*.patch; do
+  if [ -f "$file" ]; then
+    echo applying patch: $file;
+    if ! git apply --ignore-whitespace $file; then
+      echo failed to apply patch $file 1>&2
+    fi
+  fi
+done
+
```

Comment:
That looks like a reasonable thing to include in any case - if you provide that as separate PR and have tested it I can merge that (if you can't do that yet).

Reply:
Yep. I've tested it.